basculehttp/tokenFactory.go

```diff
@@ -74,6 +75,38 @@ func (btf BasicTokenFactory) ParseAndValidate(ctx context.Context, _ *http.Reque
 	return bascule.NewToken("basic", principal, bascule.NewAttributes()), nil
 }
 
+// NewBasicTokenFactoryFromList takes a list of base64 encoded basic auth keys,
+// decodes them, and supplies that list in map form of username to password.
+// If a username is encoded in two different auth keys, it will be overwritten
+// by the last occurence of that username with a password.  If anoth
+func NewBasicTokenFactoryFromList(encodedBasicAuthKeys []string) (BasicTokenFactory, error) {
```

Comment:
This is great.  It's exactly the kind of thing you want in `bascule`.

It's more common to supply a `map[string]string` for user name/password pairs.  Base64 encoding read from configuration files doesn't really offer any security anyway.  I would provide another function like `NewBasicTokenFactoryFromMap` in addition to this one.  No encoding necessary: just plaintext user names and passwords.